libraries/chain/asset_evaluator.cpp

```diff
@@ -251,6 +251,23 @@ void_result asset_fund_fee_pool_evaluator::do_apply(const asset_fund_fee_pool_op
    return void_result();
 } FC_CAPTURE_AND_RETHROW( (o) ) }
 
+static void validate_new_issuer( const database& d, const asset_object& a, account_id_type new_issuer )
```

Comment:
What's the `static` for?

Reply:
That avoids exposing the symbol outside the compilation unit.